processing/src/main/java/io/druid/query/QueryInterruptedException.java

```diff
@@ -109,6 +110,18 @@ public String getHost()
     return host;
   }
 
+  @Override
+  public String toString()
+  {
+    return StringUtils.nonStrictFormat(
```

Comment:
Ah I see, I had used that since the comment on nonStrictFormat mentioned exception messages.

Changed this to format() now